tests/conftest.py

```diff
@@ -37,7 +44,15 @@
 # (session is the pytest session, not the SQLAlchemy session)
 def pytest_sessionstart(session):
     # Will be executed before the first test
-    # _logger.debug('Initial setup fixture loaded! ')
+
+    # this is only to make the warnings into errors, so it is easier to track them down...
+    warnings.filterwarnings('error', append=True)  # comment this out in regular usage
```

Comment:
I'm probably going to comment this out once we approve the PR and ready to merge it. It is used only to verify we killed all the warnings and to help pinpoint where they came from.